README.rst

```diff
@@ -63,6 +63,9 @@ You can generate everything at the above site in your local copy of SymPy by::
     $ cd doc
     $ make html
 
+To fulfill the package requirement, ensure that all the dependencies for building
+documentation enlisted in `doc/README.rst` are satisfied.
+
```

Comment:
move to different PR

Reply:
Okay, did move both these files to #15626, Deleted those commits here. Sorry for the force push, but knew no other way.